content/blog/usethis-2-0-0/index.Rmd

```diff
@@ -65,16 +65,8 @@ These packages, in turn, might need your credentials to prove to GitHub that you
 
 If you stop reading here, just know that `gh_token_help()` is a good way to assess your credential situation:
 
-```{r, eval = FALSE}
-> gh_token_help()
-● GitHub host: 'https://github.com'
-● Host online: TRUE
-● Personal access token for 'https://github.com': '<discovered>'
-ℹ Call `gh::gh_whoami()` to see info about your token, e.g. the associated user
-ℹ To see or update the token, call `gitcreds::gitcreds_set()`
-✓ If those results are OK, you are good go to!
-ℹ Read more in the 'Managing Git(Hub) Credentials' article:
-  https://usethis.r-lib.org/articles/articles/git-credentials.html
+```{r}
+gh_token_help()
```

Comment:
I guess this is intentional?

Reply:
Yeah, that's what revealed I hadn't actually fixed the problem.